lib/client/styles/typography.scss

```diff
@@ -1,10 +1 @@
-@font-face {
-  font-family: "Inter";
-  src: url("../public/fonts/Inter.var.woff2")
-      format("woff2 supports variations"),
-    url("../public/fonts/Inter.var.woff2") format("woff2-variations");
-  font-weight: 1 999;
-  font-stretch: 75% 100%;
-  font-style: oblique 0deg 20deg;
-  font-display: swap;
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
Using Google Font is risky for the privacy of people viewing the site; is there an alternative?

Reply:
The entire site uses google analytics and google adverts anyway, not sure there's any avoiding google. Also when I downloaded this PR, it didn't work, however this does.

Reply:
Ah, alright. Fixing the Inter font issue might be worth investigating eventually due to page speed but for now I see no problem with importing Google Fonts given the site already has Google embedded into it.